blk/components/virt.c

```diff
@@ -174,7 +174,7 @@ void init(void)
     for (int i = 0; i < BLK_NUM_CLIENTS; i++) {
         blk_req_queue_t *curr_req = blk_virt_cli_req_queue(blk_client_req_queue, i);
         blk_resp_queue_t *curr_resp = blk_virt_cli_resp_queue(blk_client_resp_queue, i);
-        uint32_t queue_size = blk_virt_cli_queue_size(i);
+        uint32_t queue_size = blk_virt_cli_queue_capacity(i);
         blk_queue_init(&clients[i].queue_h, curr_req, curr_resp, queue_size);
```

Comment:
If you're changing the function to queue_capacity, shouldn't you also change the variable to queue_capacity?

Reply:
Nice catch, fixed.